wiki/osu!api/id.md

```diff
@@ -1,16 +1,15 @@
----
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
----
-
 # osu!api
 
-osu!api dapat digunakan oleh siapapun selama mereka memiliki akun osu!, berbeda dengan perizinan API pada sebelumnnya yang hanya dapat digunakan sesuai dengan permintaan kerena bersifat tertutup.
+osu! memiliki [API](https://en.wikipedia.org/wiki/API) publik yang dapat digunakan oleh siapa saja untuk memperluas dan mendukung plaform osu!. Dulunya API ini bersifat privat, dan hanya boleh digunakan jika kamu mengajukan permintaan akses, tapi sekarang siapa pun dapat menggunakan API ini selama mereka memiliki akun osu!.
```

Comment:
Mungkin halaman wikipedianya bisa diganti ke bahasa Indonesia (kalo mau)

https://id.wikipedia.org/wiki/Antarmuka_pemrograman_aplikasi

Reply:
Di versi bahasa inggris kalimat pertamanya *osu! has a public API that third-party services can use*. Mungkin bisa diganti jadi 'dapat digunakan oleh layanan pihak ketiga'.